### ISSUE TITLE
createEffect doesn't resolve void Params

### ISSUE THREAD

**Issue by Guck111** [effector/effector#404]
I have this code:

```
export const getProviders = createEffect("getProviders", {
  handler() {
    return fetch(`http://localhost:8000/api/v1/constructor/providers/`)
  },
})

getProviders()
```

and I got: ```Expected 1 arguments, but got 0.```

version: 21.5.0

**Comment by zerobias** [comment]
This is not the most actual way of creating effect, it really does not support that yet, this is a bug. The bug was overlooked because variants [with a function](https://effector.now.sh/docs/api/effector/createEffect#createeffecthandler) or [with configuration](https://effector.now.sh/docs/api/effector/createEffect#createeffect-handler-name-) are used much more often, and they indeed support handling void params:

```typescript
import { createEffect } from 'effector'

export const getProviders = createEffect(() => {
  return fetch(`http://localhost:8000/api/v1/constructor/providers/`)
})

export const getProvidersWithConfig = createEffect({
    handler() {
      return fetch(`http://localhost:8000/api/v1/constructor/providers/`)
    },
    name: 'getProvidersWithConfig'
})

getProviders()
getProvidersWithConfig()

```

[Try it in typescript repl](https://www.typescriptlang.org/play?#code/JYWwDg9gTgLgBAbzgYygUwIYzQUQGZ5rLwC+ceUEIcA5GgUTNDQLABQ7aAHpLChADsAzvADmaGAAVKAN2AATNFCFwAvCnRZcDYgApdASjUA+ROzhx0MAK5QB5CcgAWugAZOYMMAC4A9L4AbCGQMAKcIEW8ADgAGON8MMGBfGQBGX2RBEShrYmhfMFkFJSFfVwN2EgqONm5eeEzhMQlpCDlFZQB1YBgnAGFBPGBRNQ1MbHxCPQRzCzgnDAF5AKVDMzY5uatbe0IYZzcPLz9A4NDwyNj4xOS0jKyYHLyoAqKO0vLZixIAGi+4AQYEBoby0cRSN4lbq9AYCIaiVhsKrsdjg1rtEqGVEtSFdHr9QbDLFsIA)

So I suggest using one of them

Anyway, we'll try to fix this variant as well

**Comment by zerobias** [comment]
Fixed in [effector 21.6.0](https://github.com/effector/effector/releases/tag/effector%4021.6.0)
